### PR TITLE
NNS1-3269: Replaces "hardware wallet" with "Ledger device"

### PR DESCRIPTION
**Pull request by yhabib**
# Motivation

We want to avoid using the term "hardware wallet" in the app since the NNS-dapp only supports Ledgers. This initial PR focuses on replacing the term in the content. Subsequent PRs will address its removal from the codebase.

Deployed [here](https://qsgjb-riaaa-aaaaa-aaaga-cai.yhabib-ingress.devenv.dfinity.network)

# Changes

- Replaces the "hardware wallet" with "Ledger device"

# Tests

- Manually tested
- Updated UI related tests

# Todos

- [ ] Add entry to changelog (if necessary).

### CODE REVIEW

File: frontend/src/lib/i18n/en.json [hunk 1]
```diff
@@ -197,29 +197,29 @@
     "add_account": "Add Account",
     "new_account_title": "New Account",
     "new_linked_subtitle": "Create a new account controlled by your Internet Identity",
-    "attach_hardware_title": "New Account (Hardware Wallet)",
-    "attach_hardware_subtitle": "Create a new account controlled by your hardware wallet",
-    "attach_hardware_name_placeholder": "Hardware Wallet Name",
+    "attach_hardware_title": "New Account (Ledger device)",
```
Comment (csepreghy) [inline frontend/src/lib/i18n/en.json:200]: "Ledger controlled"

File: frontend/src/lib/i18n/en.json [hunk 2]
```diff
@@ -854,32 +854,32 @@
...
-    "connect_no_device": "No hardware wallet found. Is the hardware wallet connected and unlocked?",
-    "connect_many_apps": "Cannot connect to the hardware wallet. Please close all other wallet applications (e.g. Ledger Live) and try again.",
-    "connect_not_supported": "Either you have other wallet applications open (e.g. Ledger Live), or your browser doesn't support WebHID, which is necessary to communicate with your Ledger hardware wallet. Supported browsers: Chrome (Desktop) v89+, Edge v89+, Opera v76+. Error: $err",
-    "unexpected_wallet": "Found unexpected public key. Are you sure you're using the right hardware wallet?",
+    "connect_no_device": "No Ledger device found. Is the Ledger device connected and unlocked?",
+    "connect_many_apps": "Cannot connect to the Ledger device. Please close all other wallet applications (e.g. Ledger Live) and try again.",
+    "connect_not_supported": "Either you have other wallet applications open (e.g. Ledger Live), or your browser doesn't support WebHID, which is necessary to communicate with your Ledger Ledger device. Supported browsers: Chrome (Desktop) v89+, Edge v89+, Opera v76+. Error: $err",
```
Comment (dskloetd) [inline frontend/src/lib/i18n/en.json:867]: now it says "Ledger Ledger device"

File: frontend/src/lib/i18n/en.json [hunk 3]
```diff
@@ -854,32 +854,32 @@
...
+    "unexpected": "There was an error trying to attach the Ledger device.",
+    "connect": "You need to connect your Ledger device first.",
     "no_name": "No account name provided.",
-    "no_identity": "No identity connected to your hardware wallet.",
-    "already_registered": "The hardware wallet is already registered with another account.",
-    "limit_exceeded": "The limit of hardware wallet that can be attached has been reached.",
-    "register_hardware_wallet": "Error registering hardware wallet",
+    "no_identity": "No identity connected to your Ledger device.",
+    "already_registered": "The Ledger device is already registered with another account.",
+    "limit_exceeded": "The limit of Ledger device that can be attached has been reached.",
```
Comment (dskloetd) [inline frontend/src/lib/i18n/en.json:881]: A word seems to be missing here?
"number of Ledger devices" or something?

File: frontend/src/lib/i18n/en.json
Comment (dskloetd) [inline frontend/src/lib/i18n/en.json:1]: Please add a change log entry.

**Review by dskloetd — APPROVED** [review]

File: frontend/src/lib/i18n/en.json (hunk 3 above)
Comment (yhabib) [reply frontend/src/lib/i18n/en.json:881]: Maybe `the maximum number of Ledger devices that can be attached has been reached.`, what do you think @csepreghy ?

Comment (csepreghy) [reply frontend/src/lib/i18n/en.json:881]: If you capitalize the first letter, it works for me. So:
`The maximum number of Ledger devices that can be attached has been reached.`

**Review by dskloetd — APPROVED** [review]

**Pull request merged**